### PR TITLE
[AIRFLOW-XXXX] Fix typos and broken links in development docs

### PR DESCRIPTION
**Pull request by xinbinhuang**
---
Issue link: `Document only change, no JIRA issue`

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: TESTING.rst [hunk 1]
```diff
@@ -278,7 +278,7 @@ Local and Remote Debugging
 ==========================
 
 One of the great benefits of using the local virtualenv and Breeze is an option to run
-local debugging in your IDE graphical interface. You can also use ``ipdb``
+local debugging in `your IDE graphical interface <BREEZE.rst#using-your-host-ide>`__. You can also use ``ipdb``
 if you prefer `console debugging <#breeze-debugging-with-ipdb>`__.
```
Comment (xinbinhuang) [inline TESTING.rst:282]: I tried to find this broken link with `grep -R 'ipdb' .` but I could not find anything relevant. Was this rewritten into something else? Or should I remove the link if this was removed all together?

Comment (mik-laj) [reply TESTING.rst:282]: I can't find this section either.

**Review by mik-laj — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Thanks @xinbinhuang

File: TESTING.rst (hunk 1 above)
Comment (mik-laj) [reply TESTING.rst:282]: @potiuk Do you know where to find these documentation?

Comment (potiuk) [reply TESTING.rst:282]: Fixed it in https://github.com/apache/airflow/pull/7104

**Pull request merged**